tests/Helper/OutputCleaner.php

```diff
@@ -10,11 +10,7 @@ public static function cleanOutput(string $string): string
         // In the bash completion script ×2
         $string = str_replace('_sf_castor.linux-amd64.phar', '_sf_castor', $string);
         $string = str_replace('castor.linux-amd64.phar', 'castor', $string);
-        $string = preg_replace('{In functions.php line \d+:}m', 'In functions.php line XXXX:', $string);
-        $string = preg_replace('{In FunctionFinder.php line \d+:}m', 'In FunctionFinder.php line XXXX:', $string);
-        $string = preg_replace('{In ParallelHelper.php line \d+:}m', 'In ParallelHelper.php line XXXX:', $string);
-        $string = preg_replace('{In Process.php line \d+:}m', 'In Process.php line XXXX:', $string);
-        $string = preg_replace('{In ContextRegistry.php line \d+:}m', 'In ContextRegistry.php line XXXX:', $string);
+        $string = preg_replace('{In (\w+).php line \d+:}m', 'In \1.php line XXXX:', $string);
```

Comment:
Not a big fan of this as it hides a bit the correct behaviour or not of the `fix_exception()` function

Reply:
Fixed, now i replace it only for classname